src/components/variant-grid/index.js

```diff
@@ -248,98 +248,102 @@ const VariantGrid = ({ product, variants, onChange, edit, onEdit, onCopy }) => {
           </tr>
         </TableHead>
         <tbody>
-          {variants.map((v, row) => (
```

Comment:
Not a lot has changed in this file - think i moved something slightly causing the massive markings. The relevant part is the addition of a check if the variant has even been created. If not (i.e. ID === undefined) we want to hide it until it is completed.